src/components/SingleReview.jsx

```diff
@@ -0,0 +1,43 @@
+import axios from "axios";
+import { useEffect,useState } from "react";
+import { useParams } from "react-router-dom";
+import { getSingleReview } from "../utils/api";
+
+const SingleReview=()=>{
+    const[review,setReview]=useState([])
+    const{review_id}=useParams()
+    const[votes,setVotes]=useState(0)
+    const[isClicked,setIsClicked]=useState(false)
+    useEffect(()=>{
+        getSingleReview(review_id).then((rev)=>{
+            setReview(rev)
+            setVotes(rev.votes)
+        })
+    },[review_id])
+
+const handleVotes=()=>{
+    if(isClicked){
```

Comment:
You haven't interacted with the API!! If you refresh it stays on the same amounts of votes it was on before you voted. You need to use the patch endpoint we have

Reply:
Oh of course. Will do.